man/crypttab.xml

```diff
@@ -431,6 +435,15 @@
         before it is used to unlock the LUKS volume.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>try-empty-password=</option></term>
+
+        <listitem><para>Takes a boolean argument. If enabled, right before inquiring the user for a password
```

Comment:
This use of inquiring/enquiring is inverted (see https://www.oxfordlearnersdictionaries.com/definition/english/enquire for an explanation, we're not asking *about* the user). Before in the comment it doesn't matter as much, but here in the user-visible docs, let's try to do better. Please just say "ask".